test/jdk/javax/swing/JScrollPane/TestMouseWheelScroll.java

```diff
@@ -0,0 +1,126 @@
+import javax.swing.ListModel;
+import javax.swing.JScrollPane;
+import javax.swing.JFrame;
+import javax.swing.JList;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+/*
+ * @test
+ * @key headful
+ * @requires (os.family != "mac")
+ * @bug  6911375
+ * @summary Verifies mouseWheel effect on JList without scrollBar
+ */
+public class TestMouseWheelScroll {
+
+    static JFrame frame;
+    static JScrollPane scrollPane;
+    static Point p;
+    static int width;
+    static int height;
+    static volatile Point viewPosition;
+    static volatile Point newPosition;
+
+    private static void setLookAndFeel(UIManager.LookAndFeelInfo laf) {
+        try {
+            UIManager.setLookAndFeel(laf.getClassName());
+        } catch (UnsupportedLookAndFeelException ignored) {
+            System.out.println("Unsupported L&F: " + laf.getClassName());
+        } catch (ClassNotFoundException | InstantiationException
+                 | IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void main(String[] args) throws Exception {
+
+        for (UIManager.LookAndFeelInfo laf : UIManager.getInstalledLookAndFeels()) {
+            System.out.println("Testing L&F: " + laf.getClassName());
+            SwingUtilities.invokeAndWait(() -> setLookAndFeel(laf));
+            Robot robot = new Robot();
+            robot.setAutoDelay(100);
+
+            try {
+                SwingUtilities.invokeAndWait(() -> {
+                    frame = new JFrame();
+                    JList list = new JList(createListModel());
+                    // disable list bindings
+                    list.getInputMap().getParent().clear();
+                    scrollPane = new JScrollPane(list);
+
+                    scrollPane.setVerticalScrollBarPolicy(JScrollPane.VERTICAL_SCROLLBAR_NEVER);
+                    frame.add(scrollPane);
+                    frame.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
+                    frame.setSize(200,200);
+                    frame.setLocationRelativeTo(null);
+                    frame.setVisible(true);
+                });
+                robot.waitForIdle();
+                robot.delay(1000);
+                SwingUtilities.invokeAndWait(() -> {
+                    p = frame.getLocationOnScreen();
+                    width = frame.getWidth();
+                    height = frame.getHeight();
+                });
+                robot.mouseMove(p.x + width / 2, p.y + height / 2);
+                robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+                robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+                SwingUtilities.invokeAndWait(() -> {
+                    viewPosition = scrollPane.getViewport().getViewPosition();
+                });
+                robot.delay(1000);
+                robot.mouseWheel(1);
+                robot.delay(500);
```

Comment:
You don't call `waitForIdle` or `delay` after the mouse click because it just selects an item but doesn't change the position, right?

Reply:
added